data/data/nutanix/master/main.tf

```
@@ -0,0 +1,51 @@
+locals {
+  description = "Created By OpenShift Installer"
+}
+
+provider "nutanix" {
+  wait_timeout = 60
+  username     = var.username
+  password     = var.password
+  endpoint     = var.prism_central
+  insecure     = var.insecure
+  port         = var.port
+}
+
+resource "nutanix_virtual_machine" "vm_master" {
+  count = var.master_count
```

tf-fmt is failing due '=' not lining up with the rest of the block
```suggestion
  count                = var.master_count
```